packages/blockchain-wallet-v4-frontend/src/scenes/Signup/index.tsx

```diff
@@ -160,7 +146,7 @@ class SignupContainer extends React.PureComponent<
 
     return (
       <>
-        {isUserInUK && (
+        {(isUserInUK || userSelectedUK) && (
           <UKHeaderWrapper>
             <UkBanner userLoggedOut />
```

Comment:
As now the app can check if the user is in the UK with the IP without being logged in, I believe this prop is not required anymore here nor in the UkBanner

Reply:
did you mean here? https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/release/v4.94/packages/blockchain-wallet-v4-frontend/src/components/Banner/UkBanner/index.tsx#L10

I left it there because I didn't want to mess with any existing logic other than deciding user country.